native_client/README.md

```diff
@@ -10,12 +16,13 @@ To download the pre-built binaries, use `util/taskcluster.py`:
 python util/taskcluster.py --target /path/to/destination/folder
 ```
 
-If you need some binaries different than current master, like `v0.2.0-alpha.6`, you can use `--branch`:
+If you need binaries which are different than current master (e.g. `v0.2.0-alpha.6`) you can use the `--branch` flag:
+
 ```bash
 python3 util/taskcluster.py --branch "v0.2.0-alpha.6"
 ```
 
-This will download and extract `native_client.tar.xz` which includes the deepspeech binary and associated libraries as well as the custom decoder OP. `taskcluster.py` will download binaries for the architecture of the host by default, but you can override that behavior with the `--arch` parameter. See the help info with `python util/taskcluster.py -h` for more details.
+`util/taskcluster.py` will download and extract `native_client.tar.xz`.  `native_client.tar.xz` includes (1) the `deepspeech` binary, (2) associated libraries, and (3) the custom decoder TensorFlow OP. `taskcluster.py` will download binaries for the architecture of the host by default, but you can override that behavior with the `--arch` parameter. See `python util/taskcluster.py -h` for more details.
```

Comment:
I would remove mentions of the decoder OP as we no longer have that.

Reply:
removed